app/Models/Member.php

```diff
@@ -21,4 +34,14 @@ public function tasks()
     {
     	return $this->hasMany(Task::class);
     }
+
+    public function leadingProjects()
+    {
+        return $this->hasMany(Project::class, 'foreign_key', 'leader_id');
+    }
+
+    public function getIsAdminLabelAttribute()
+    {
+        return self::IS_ADMIN[$this->is_admin];
```

Comment:
sao đoạn này nó lại loằng ngoằng thế này? `return $this->is_admin` thôi không được à